File: code/mathematical-algorithms/coprime_numbers/CoprimeNumbers.java
class CoprimeNumbers{
	public static long gcd(long c,long d){
		if(d==0){
			return c;
		}else{
			return gcd(d,c%d);
		}
	}
	public static boolean isCoprime(long a,long b){
		return gcd(a,b)==1;
	}
	public static void main(String[] args){
		System.out.println(isCoprime(6,3));
	}
}

File: cosmos-master/code/mathematical-algorithms/coprime_numbers/coprime_numbers.cpp
#include <bits/stdc++.h>
// Part of Cosmos by OpenGenus Foundation
using namespace std;

int gcd(int i, int j)
{
        while(j != 0) {
                int x = i;
                i = j;
                j = x % j;
        }
        return i;
}

bool isCoPrime(int i, int j)
{
        return gcd(i,j) == 1;
}

int main()
{
        int i,j;
        cout << "Enter two numbers : ";
        scanf("%d %d", &i, &j);
        cout << isCoPrime(i,j) << endl;
        return 0;
}
